modules/apps/object/object-test/src/testFunctional/macros/ObjectAdmin.macro

```diff
@@ -423,6 +423,17 @@ definition {
 		AssertElementNotPresent(locator1 = "ObjectAdmin#VIEW_OBJECT_FIELD_LABEL");
 	}
 
+	macro assertObjectInApplicationsMenu {
+		ApplicationsMenuHelper.openApplicationsMenu();
+
+		ApplicationsMenu.gotoPanel(panel = "${panel}");
+
+		AssertElementPresent(
+			key_category = "${category}",
+			key_portlet = "${portlet}",
+			locator1 = "ApplicationsMenu#PORTLET");
+	}
+
```

Comment:
Hi @andreefarias 

You can use this macro: `assertObjectIsDisplayedOnControlPanel` that is in `objectPortlet.macro`